modules/nlu/src/backend/application/tests/app-integration.test.ts

```diff
@@ -761,4 +775,46 @@ describe('NLU API integration tests', () => {
     expect(dependencies.socket).toHaveBeenCalledWith(expectTs({ botId, status: 'needs-training' }))
     expect(dependencies.socket).not.toHaveBeenCalledWith(expectTs({ botId, status: 'training-pending' }))
   })
+  test('Training queue starts out paused and does not start any training', async () => {
+    // arrange
+    const lang = 'en'
+
+    const definitions = makeBaseDefinitions([lang])
+
+    const modelId = modelIdService.makeId({
+      ...definitions,
+      languageCode: lang,
+      seed: nluSeed,
+      specifications: specs
+    })
+
+    const fileSystem = {
+      [botId]: {
+        definitions,
+        modelsOnFs: [modelId]
+      }
+    }
+
+    const core = { languages: [lang], specs }
+    const dependencies = makeDependencies(core, fileSystem)
+
+    const app = makeApp(dependencies)
+    const engineTrainSpy = jest.spyOn(dependencies.engine, 'train')
+
+    // act
+    await app.initialize()
+
+    await app.mountBot({
+      id: botId,
+      defaultLanguage: lang,
+      languages: [lang]
+    })
+
+    expect(engineTrainSpy).not.toHaveBeenCalled()
+    await sleep(10)
+    await app.resumeTrainings()
+
+    expect(engineTrainSpy).not.toHaveBeenCalled()
```

Comment:
mmmhhhh why should `engine.train()` not be called ? I feel like it should be called:

```ts
await waitForTrainingsToBeDone(app)
expectTrainingToStartAndComplete(socket, trainingRepo, { botId, language: lang })
// expect training to have occured (just look a some tests above)
```

Reply:
Yeah ok that test did not make any sense, fixed now!